### DISCUSSION TITLE
docker bin/kvrocks 354M too large?

### DISCUSSION THREAD

**Discussion by segment11** [apache/kvrocks#1089] [Q&A] [answered]
-rwxr-xr-x. 1 root root 354M Nov 10 17:33 kvrocks*

Hi，is it normal?

**Comment by PragmaTwice** [comment, marked as answer]
Hi @segment11 , thanks for your question!

In order to be able to debug easier, we keep all the debug information in the binary, which makes the binary larger (~ several hundred MiB).

I do not think it is a big deal, since the hard drives nowadays are huge in capacity. :rofl:

**Reply by PragmaTwice** [reply]
But of course, if you want a small binary (maybe several ten MiB), you can build kvrocks by your own, and pass `-DCMAKE_BUILD_TYPE=Release` to it, e.g.
```sh
./x.py build -DCMAKE_BUILD_TYPE=Release
```

**Reply by segment11** [reply]
Thanks a lot
